iModelCore/ECPresentation/Source/Shared/ValueHelpers.cpp

```diff
@@ -5,6 +5,7 @@
 #include <ECPresentationPch.h>
 #include "ValueHelpers.h"
 #include "../RulesEngineTypes.h"
+#include "../Rules/PresentationRuleJsonConstants.h"

```

Comment:
if we're using these constants here, then I think they should be moved to `ValueHelpers.h`